### PR TITLE
improve japanese locale.

### PR DESCRIPTION
**Pull request by hATrayflood**
- translate new strings.
- unify `,` to `、`
- unify `部屋` to `ルーム`
- unify `オブジェクトをピン留め` to `オブジェクトを固定`
- keep english brand names.
- remove unnecessary inserted spaces like `\n  \n 続行` to `\n\n続行`
- restore deleted spaces like `Bingで検索|` to `Bingで検索 | `
- fix other unnatual japanese translations.

### CODE REVIEW

File: src/assets/locales/ja.json [hunk 1]
```diff
@@ -427,42 +427,42 @@
...
   "tweet-dialog.posted": "あなたのツイートは投稿されました。",
-  "tweet-dialog.close": "close",
+  "tweet-dialog.close": "閉じる",
   "oauth-dialog.sign-in.twitter": "Twitterに接続",
   "oauth-dialog.sign-in.discord": "Discordにサインイン",
   "oauth-dialog.sign-in.slack": "Slackにサインイン",
-  "oauth-dialog.message.twitter": "Twitterに接続して,%app-name%からツイートを送信してください。",
-  "cloud.primary_tagline": "Hubs Cloudは,デスクトップ,モバイル,またはVRヘッドセットからアクセスできるプライベートな\n collaborative3Dルーム用のサーバーインフラストラクチャを展開します。",
+  "oauth-dialog.message.twitter": "Twitterに接続して、%app-name%からツイートを送信してください。",
+  "cloud.primary_tagline": "Hubs Cloudは、デスクトップ、モバイル、またはVRヘッドセットからアクセスできるプライベートな\ncollaborative 3Dルーム用のサーバーインフラストラクチャを展開します。",
```
Comment (kn1cht) [inline src/assets/locales/ja.json:453]: "collaborative" also should be translated :
「Hubs Cloudは、デスクトップ、モバイル、またはVRヘッドセットからアクセスできるプライベートでコラボレーション可能な3Dルーム用のサーバーインフラストラクチャを展開します。」

**Review by kn1cht — CHANGES_REQUESTED** [review]
Translation texts look mostly natural to me. Thanks for your great work!

File: src/assets/locales/ja.json [hunk 2]
```diff
@@ -303,20 +303,20 @@
...
   "preferences.enableOnScreenJoystickRight": "画面上の右のジョイスティックで見回せるようにする",
-  "preferences.enableGyro": "Enable gyroscope (when supported by browser/device)",
-  "preferences.invertTouchscreenCameraMove": "タッチスクリーンのカメラ移動の方向を反転",
+  "preferences.enableGyro": "ジャイロスコープを有効にする（ブラウザーとデバイスがサポートしている場合）",
+  "preferences.invertTouchscreenCameraMove": "タッチスクリーンのカメラ移動の方向を反転する",
   "preferences.browserDefault": "ブラウザのデフォルト",
   "preferences.locale": "言語",
-  "preferences.onlyShowNametagsInFreeze": "フリーズ中の名前タグのみを表示",
-  "preferences.animateWaypointTransitions": "ウェイポイントトランジションをアニメーション化",
+  "preferences.onlyShowNametagsInFreeze": "フリーズ中の名前タグのみを表示する",
```
[the remark this answers is not in the document]
Comment (hATrayflood) [reply src/assets/locales/ja.json]: thanks! updated https://github.com/mozilla/hubs/pull/3422/commits/8843b758493fd953647924fa5c55861165b8c0d7

File: src/assets/locales/ja.json (hunk 1 above)
Comment (hATrayflood) [reply src/assets/locales/ja.json]: thanks! updated https://github.com/mozilla/hubs/pull/3422/commits/8843b758493fd953647924fa5c55861165b8c0d7

**Review by kn1cht — APPROVED** [review]

**Comment by brianpeiris** [comment]
I will go ahead and merge this now. Thank you!

**Pull request merged**